tools/symflower.go

```diff
@@ -3,132 +3,82 @@ package tools
 import (
 	"errors"
 	"fmt"
-	"os"
-	"os/exec"
 	"path/filepath"
 	"regexp"
 	"runtime"
 	"strconv"
 	"strings"
-	"time"
 
 	pkgerrors "github.com/pkg/errors"
-	"github.com/symflower/lockfile"
-	"github.com/zimmski/osutil"
-
 	"github.com/symflower/eval-dev-quality/log"
 	"github.com/symflower/eval-dev-quality/util"
+	"github.com/zimmski/osutil"
 )
 
+// symflower holds the "Symflower" tool.
+type symflower struct{}
+
+// Symflower is the "Symflower" tool.
+var Symflower Tool = &symflower{}
```

Comment:
No singletons. It is an anti pattern. Just do a NewSymflower() function, and use it where you need it. the struct is small enough to be thrown away right after use.